proposals/simd/SIMD.md

```diff
@@ -304,6 +304,25 @@ def S.shuffle(a, b, s):
     return result
```

Comment:
Is it the validation error implied in the `b[out-of-range]`  access ? Or will this be a run-time error ?

Reply:
As per comments above wrt LaneIdx32 specifying 0-31 indices, this would be a validation error - similar to all other cases where LaneIdxN is an input, this is currently not explicitly clarified in the spec but I'm assuming it will be listed once when describing LaneIdx encoding, not in each instruction that uses these.

Reply:
Thanks @zeux, for some reason I can't mark these issues as "resolved" but I think this is good to go as well (cc @dtig @tlively  - maybe one has to enable that in the repo? also sorry that I missed the meeting yesterday, the invitation came pretty late and the meeting was too late for me in CET time).

Reply:
I checked settings, but there doesn't seem to be anything I can change to enable resolving here. Maybe @binji knows? Sorry the meeting time did not work for you, I usually check the day before for any new attendees and I may have just missed that based on when the form was filled out. For now, I'm going to go ahead and merge as it looks like all the outstanding issues have been addressed.

Reply:
Not sure what permission it is, but I can resolve here.